tutorials/light_config.md

```diff
@@ -0,0 +1,22 @@
+\page light_config Light config
+
+This tutorial gives an introduction to the Ignition Gazebo's service `/world/<world name>/light_config`.
+This service will allow to modify lights in the scene.
```

Comment:
should we give some simple steps of building and running the example? e.g. `cd` into `standalone/light_control`, build the code, run `ign-gazebo empty.sdf` and then `./light_control` executable, etc